cli/src/base/ContentDirectoryCommandBase.ts

```diff
@@ -322,7 +322,7 @@ export default abstract class ContentDirectoryCommandBase extends RolesCommandBa
   async parseToKnownEntityJson<T>(entity: Entity): Promise<FlattenRelations<T>> {
     const entityClass = (await this.classEntryByNameOrId(entity.class_id.toString()))[1]
     return (_.mapValues(this.parseEntityPropertyValues(entity, entityClass), (v) =>
-      v.value.toJSON()
+      v.type !== 'Single<Bool>' && v.value.toJSON() === false ? null : v.value.toJSON()
```

Comment:
Adding some `(` brackets `)` well go a long way to help understand the logic of this expression

Reply:
I added them by they are automatically stripped due to linter/prettier rules

Reply:
yes I was expecting that hehe. Okay now with your clarification in https://github.com/Joystream/joystream/pull/1614#issuecomment-721551034 it makes sense and confirms my memory of logical operator precedence is still intact ;)